src/pages/StoryPage.tsx

```diff
@@ -3,7 +3,7 @@ import Helmet from 'react-helmet-async'
 import { getBlockComponent } from '../blocks'
 import { StoryContainer } from '../storyblok/StoryContainer'
 
-export const StoryPage: React.FunctionComponent = () => (
+export const StoryPage: React.SFC = () => (
```

Comment:
Any particular reason for this? SFC is deprecated

Reply:
Ah, my bad

Reply:
so easy to miss haha 😄 i just knew because my editor screams at me because it's annotated with `@deprecated`